pkg/gadgettracermanager/gadgettracermanager.go

```diff
@@ -267,6 +268,25 @@ func (g *GadgetTracerManager) RemoveContainer(ctx context.Context, containerDefi
 	return &pb.RemoveContainerResponse{}, nil
 }
 
+func (g *GadgetTracerManager) LookupMntnsByContainer(namespace, name, container string) (ret []uint64) {
+	for _, c := range g.containers {
+		if namespace != c.Namespace {
+			continue
+		}
+		if name != c.Podname {
+			continue
+		}
+		if container != "" && container != c.ContainerName {
```

Comment:
I think we should document that passing `container = ""` will return the namespaces for **all** containers in the pod.

Reply:
Also, what about having two different methods?: 
- `LookupMntnsByContainer(namespace, pod, container string) uint64`: requires all three parameters and returns a single mount namespace id. 
- `LookupMntnsByPod(namespace, pod string) []uint64`: returns the mount namespace ids for all containers in the pod.